### PR TITLE
Adding test for activation function Mish

### PR DESCRIPTION
**Pull request by ma-hei**
Adding a unit test for activation function Mish.
Testing non-monotonicity property of Mish and
Mish's derivative.

### CODE REVIEW

File: operations_test.go [hunk 1]
```diff
@@ -348,6 +348,58 @@ func TestGt(t *testing.T) {
 
 }
 
+func GetF64 (n *Node) float64 {
```
Comment (ma-hei) [inline operations_test.go]: Is there a simpler way to do this? I need to get the value of the node in order to compare it with assert.Greater/assert.Less.

File: operations_test.go [hunk 2]
```diff
@@ -348,6 +348,58 @@ func TestGt(t *testing.T) {
...
+
+	Let(x0, -2.5)
+	Let(x1, -2.2)
+	Let(x2, 1.0)
+
+	f0 = Must(Mish(x0))
+	f1 = Must(Mish(x1))
+	f2 = Must(Mish(x2))
+
+	if grad0, err = Grad(f0, x0); err != nil {
```
Comment (ma-hei) [inline operations_test.go]: I'm repeating myself here. Maybe there's a better way?

File: operations_test.go [hunk 3]
```diff
@@ -697,6 +749,7 @@ func TestMean(t *testing.T) {
 	}
 }
 
+
```
Comment (ma-hei) [inline operations_test.go]: I'll remove this new line.

**Comment by chewxy** [comment]
Lightning storm melted my computer's power cable into motherboard. Will review later on a different machine (but browsing via phone indicates it looks good)

File: operations_test.go (hunk 1 above)
Comment (chewxy) [reply operations_test.go]: https://github.com/gorgonia/gorgonia/blob/master/testsetup_test.go#L53

**Comment by chewxy** [comment]
@ma-hei  all good! Those two comments are minor issues. You can fix them if you want (or I can just merge this if you aren't free). More importantly, can you please email me? chewxy [at] gmail. More via email

**Comment by ma-hei** [comment]
I removed the unnecessary new line and used the existing "extractF64" method. Feel free to merge! Sure, I'll send an email later today!

**Pull request merged**